packages/documentation-site/patternfly-docs/content/layouts/compass/cards/AnimationsOverviewClusterInventory.tsx

```diff
@@ -0,0 +1,126 @@
+import { useState } from "react";
+import {
+  Card,
+  CardHeader,
+  CardTitle,
+  CardBody,
+  CardFooter,
+  Dropdown,
+  DropdownList,
+  DropdownItem,
+  MenuToggle,
+  Flex,
+  FlexItem,
+  Button,
+  Icon,
+  CompassPanel,
+} from "@patternfly/react-core";
+
+import ListIcon from "@patternfly/react-icons/dist/esm/icons/list-icon";
+import EllipsisVIcon from "@patternfly/react-icons/dist/esm/icons/ellipsis-v-icon";
+import CubesIcon from "@patternfly/react-icons/dist/esm/icons/cubes-icon";
+import CubeIcon from "@patternfly/react-icons/dist/esm/icons/cube-icon";
+import ArrowRightIcon from "@patternfly/react-icons/dist/esm/icons/arrow-right-icon";
+
+export const ClusterInventoryCard: React.FunctionComponent = () => {
+  const [isKebabOpen, setIsKebabOpen] = useState(false);
+
+  // Data for the list items in the card body
+  const inventoryItems = [
+    { icon: <CubesIcon />, text: "15 Deployments" },
+    { icon: <CubesIcon />, text: "15 Deployments" },
+    { icon: <CubesIcon />, text: "15 Deployments" },
+    { icon: <CubesIcon />, text: "15 Deployments" },
+    { icon: <CubeIcon />, text: "126 pods" },
+  ];
+
+  // Items for the kebab dropdown menu
+  const dropdownItems = [
+    <DropdownItem key="action1">Action 1</DropdownItem>,
+    <DropdownItem key="action2">Action 2</DropdownItem>,
+    <DropdownItem key="action3">Action 3</DropdownItem>,
+  ];
+
+  // The kebab menu toggle button
+  const kebabToggle = (toggleRef: React.Ref<any>) => (
+    <MenuToggle
+      ref={toggleRef}
+      variant="plain"
```

Comment:
Can you add `isCircle` here to make it a pill shape?

```suggestion
      variant="plain"
      isCircle
```